docs/application/native/guides/multimedia/stream-recorder.md

```diff
@@ -134,7 +134,9 @@ To enable your application to use the stream recorder functionality:
 
 To get and set information about video and audio encoding:
 
-1. To get a list of video codecs your device supports, call the `streamrecorder_foreach_supported_video_encoder()` function. One of its parameters is a callback, which is called for each codec supported for the given stream recorder.In the following example, the codec of the stream recorder is set to the first found supported codec.
+1. To get a list of video codecs your device supports, call the `streamrecorder_foreach_supported_video_encoder()` function. One of its parameters is a callback, which is called for each codec supported for the given stream recorder.
```

Comment:
line 99: Add a space. (in mobile and wearableapplications)

Reply:
done